torch/_refs/__init__.py

```diff
@@ -1045,6 +1046,50 @@ def sum(
         output_dtype_kind=REDUCTION_OUTPUT_TYPE_KIND.SAME,
     )
 
+def amin(
+    a: Tensor,
+    dim: Union[Optional[int], Optional[List[int]]] = None,
+    keepdim: bool = False,
+    *,
+    dtype=None,
```

Comment:
I don't think PyTorch amin has a dtype kwarg, right? Are we going to unify dtype kwargs on all reduction ops?

Reply:
Oh you are right, let me fix the ref.

Reply:
Other than that, looks identical to the ATen operator.

Reply:
That's because for aten amin/amax the overloads are actually sane.